nexuse2e-webapp/src/main/frontend/src/styles.scss

```diff
@@ -188,13 +188,19 @@ a {
     }
 
     .label {
-      width: 120px;
+      width: 200px;
       font-weight: bold;
       vertical-align: baseline;
       word-break: break-word;
       white-space: normal;
       text-align: left;
     }
+
+    @media (max-width: $mobileBreakpoint) {
```

Comment:
You can use the $mobileBreakpointForMaxWidth here, so it breaks at 500, not 501 pixels.